Missing stdint stuff on CentOS 7

**Describe the issue**
A clear and concise description of what the bug is.

```
In file included from /scratch/mtraudt/shadow/src/main/host/descriptor/tcp_retransmit_tally.cc:1:0:
/scratch/mtraudt/shadow/src/main/host/descriptor/tcp_retransmit_tally.h:3:30: fatal error: bits/stdint-intn.h: No such file or directory
 #include <bits/stdint-intn.h>
```

**To Reproduce**
Steps to reproduce the behavior. If you problem involves a simulation, please include the configuration file that you used.

```
./setup build
```

**Operating System (please complete the following information):**

CentOS Linux release 7.6.1810 (Core)

Linux koios1.chacs.nrl.navy.mil 4.4.180-2.el7.elrepo.x86_64 #1 SMP Wed May 22 08:01:41 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux

**Shadow (please complete the following information):**

Doesn't compile, but HEAD is 9f7cd4131876f5a6df0b851b15dbe3180210bfec

 - Which plug-ins you are using:  N/A

**Additional context**

1d61d4bed7ede64ee006ad49cc17335705dcd0e9 seems to be the offending commit. At least, it's the one that last touched the `#include` lines with `bits/`.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,6 +15,9 @@ message(STATUS "System processor: ${CMAKE_SYSTEM_PROCESSOR}")
 ## ensure cmake version
 cmake_minimum_required(VERSION 2.8.8 FATAL_ERROR)
 
+## building with support for C11 on platforms that default to C99 or C89
+set (CMAKE_C_FLAGS "-std=gnu11 ${CMAKE_C_FLAGS}")
+
 ## ensure unix environment (CMAKE_SYSTEM_NAME == "Linux")
 if((NOT UNIX) OR (NOT (CMAKE_SYSTEM_NAME STREQUAL "Linux")))
     message(FATAL_ERROR "Shadow requires a Unix/Linux environment.")

--- a/src/main/core/slave.c
+++ b/src/main/core/slave.c
@@ -3,8 +3,6 @@
  * See LICENSE for licensing information
  */
 
-#include <bits/types/struct_rusage.h>
-#include <bits/types/struct_timeval.h>
 #include <errno.h>
 #include <glib.h>
 #include <netinet/in.h>

--- a/src/main/host/descriptor/tcp.c
+++ b/src/main/host/descriptor/tcp.c
@@ -7,7 +7,6 @@
 
 #include "main/host/descriptor/tcp.h"
 
-#include <bits/stdint-uintn.h>
 #include <errno.h>
 #include <math.h>
 #include <netinet/tcp.h>

--- a/src/main/host/descriptor/tcp_retransmit_tally.h
+++ b/src/main/host/descriptor/tcp_retransmit_tally.h
@@ -1,7 +1,5 @@
 #ifndef SHD_TCP_RETRANSMIT_TALLY_H_
 #define SHD_TCP_RETRANSMIT_TALLY_H_
-#include <bits/stdint-intn.h>
-#include <bits/stdint-uintn.h>
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>

--- a/src/main/host/descriptor/timer.c
+++ b/src/main/host/descriptor/timer.c
@@ -5,9 +5,6 @@
 
 #include "main/host/descriptor/timer.h"
 
-#include <bits/types/struct_itimerspec.h>
-#include <bits/types/struct_timespec.h>
-#include <bits/types/time_t.h>
 #include <errno.h>
 #include <string.h>
 #include <sys/time.h>

--- a/src/main/host/host.c
+++ b/src/main/host/host.c
@@ -4,8 +4,6 @@
  * See LICENSE for licensing information
  */
 
-#include <bits/stdint-uintn.h>
-#include <bits/types/struct_timeval.h>
 #include <errno.h>
 #include <glib.h>
 #include <math.h>

--- a/src/main/host/process.c
+++ b/src/main/host/process.c
@@ -10,13 +10,6 @@
 #ifndef __USE_GNU
 #define __USE_GNU 1
 #endif
-#include <bits/stdint-intn.h>
-#include <bits/stdint-uintn.h>
-#include <bits/types/clockid_t.h>
-#include <bits/types/struct_timespec.h>
-#include <bits/types/struct_timeval.h>
-#include <bits/types/struct_tm.h>
-#include <bits/types/time_t.h>
 #include <dlfcn.h>
 #include <errno.h>
 #include <fcntl.h>

--- a/src/main/host/process.h
+++ b/src/main/host/process.h
@@ -11,9 +11,6 @@
 #define _GNU_SOURCE
 #endif
 
-#include <bits/stdint-intn.h>
-#include <bits/types/clockid_t.h>
-#include <bits/types/time_t.h>
 #include <glib.h>
 #include <stdarg.h>
 #include <stdint.h>

--- a/src/support/logger/logger.c
+++ b/src/support/logger/logger.c
@@ -3,6 +3,7 @@
 #include <pthread.h>
 #include <stdbool.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/time.h>
 
 static Logger* defaultLogger = NULL;

--- a/src/test/test_glib_helpers.h
+++ b/src/test/test_glib_helpers.h
@@ -3,6 +3,7 @@
 
 #include <errno.h>
 #include <glib.h>
+#include <string.h>
 #include <unistd.h>
 
 #define assert_true_errstring(c, s)                                            \

--- a/src/test/udp/test_udp.c
+++ b/src/test/udp/test_udp.c
@@ -8,6 +8,7 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <sys/socket.h>
 #include <sys/stat.h>
 #include <sys/types.h>

--- a/src/test/udp/test_udp_uniprocess.c
+++ b/src/test/udp/test_udp_uniprocess.c
@@ -8,6 +8,7 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <sys/socket.h>
 #include <sys/stat.h>
 #include <sys/types.h>

--- a/src/test/unistd/test_unistd.c
+++ b/src/test/unistd/test_unistd.c
@@ -2,6 +2,7 @@
 #include <signal.h>
 #include <stdbool.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/utsname.h>
 #include <unistd.h>
 

